### DISCUSSION TITLE
Can I simule real browser?

### DISCUSSION THREAD

**Discussion by damidoug** [tlsfuzzer/tlslite-ng#518] [General]
How can I set the chrome fingerprint, make the handshake and integrate it on httpx?
in this [raw1](https://raw.githubusercontent.com/bogdanfinn/tls-client/master/profiles/internal_browser_profiles.go), [raw2](https://raw.githubusercontent.com/lwthiker/curl-impersonate/main/tests/signatures/chrome.yaml) have the chrome fingerprint config using in go.

thanks

**Comment by tomato42** [comment]
Look into HandshakeSettings, but it might not be flexible enough to mimic Chrome. I'm pretty sure we don't support some of the extensions Chrome does so mimicking it exactly may not be possible. I use tlsfuzzer when I need to mimic a client exactly.

**Reply by damidoug** [reply]
Can you show me some example using tlsfuzzer to mimic chrome?

**Reply by tomato42** [reply]
https://github.com/tlsfuzzer/tlsfuzzer/blob/master/scripts/test-client-compatibility.py
